nodecontrol/algodControl.go

```diff
@@ -40,6 +40,22 @@ const StdErrFilename = "algod-err.log"
 // StdOutFilename is the name of the file in <datadir> where stdout will be captured if not redirected to host
 const StdOutFilename = "algod-out.log"
 
+type NodeAlreadyStoppedError struct {
```

Comment:
Travis failure: Comment is missing here.
`nodecontrol/algodControl.go:43:6: exported type NodeAlreadyStoppedError should have comment or be unexported
`